### ISSUE TITLE
Experiment with larger arrow on mobile auction page 

### ISSUE THREAD

**Issue by bcjgit** [nounsDAO/nouns-monorepo#362]

**Comment by hippolyte42** [comment]
before:
![image](https://user-images.githubusercontent.com/25116785/177058614-286fe614-3fed-4133-bb32-9f6d46fc3912.png)
after:
![image](https://user-images.githubusercontent.com/25116785/177058598-0cde1c91-efb4-4361-89a4-b1964aa33979.png)

In `packages/nouns-webapp/src/components/AuctionNavigation/AuctionNavigation.module.css` add
```
@media (max-width: 844px) {
  .leftArrowWarm,
  .leftArrowCool,
  .rightArrowWarm,
  .rightArrowCool {
    width: 2.5rem;
    height: 2.5rem;
  }
}
```

In `packages/nouns-webapp/src/components/AuctionActivityDateHeadline/AuctionActivityDateHeadline.module.css` add
```
@media (max-width: 844px) {
  .wrapper {
      margin-left: 6rem;
    }
  .date {
    line-height: 35px;
  }
}
```

Let me know if this solves the issue!

**Comment by hippolyte42** [comment]
> Hey! Thanks for putting up this PR. One question ... is it just me or do these arrows not look the same as the ones we currently have in prod? They look thinner to me ....

No, the [before screenshot](https://user-images.githubusercontent.com/25116785/177058614-286fe614-3fed-4133-bb32-9f6d46fc3912.png) is exactly what prod looks like on Iphone 12...

How should we proceed now? I would love to create my own branch & merge request for this issue, if it is allowed.
